templates/theories/topology.md

```diff
@@ -90,6 +90,18 @@ If `X` is a topological space, and `x ∈ X`, then the _neighbourhood filter_ `
 
 Why are we interested in these filters? Well, given a map `f` from `ℕ` to a topological space `X`, one can check that the resulting sequence `f 0`, `f 1`, `f 2`... tends to `x ∈ F` if and only if the pre-image of any element in the filter `𝓝 x` is in the cofinite filter on `ℕ` -- this is just another way of saying that given any open set `U` containing `x`, there exists `N` such that for all `n ≥ N`, `f n ∈ U`. So filters provide a way of thinking about limits.
 
+As an example, below are three limits formulated in Lean.
+The example uses the filters `at_top` and `at_bot` that represent "tends to `∞`" and "tends to `-∞`" in a type equipped with an order.
+```lean
+-- The limit of 2 * x tends to 6 as x goes to 3
```

Comment:
```
-- The limit of 2 * x as x tends to 3 is 6 
```
To quibble, the limit "is" rather than "tends to".

Reply:
Thanks!